src/BuiltInTools/dotnet-watch/LaunchBrowserFilter.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.DotNet.Watcher.Tools
 {
     public sealed class LaunchBrowserFilter : IWatchFilter, IAsyncDisposable
     {
-        private static readonly Regex NowListeningRegex = new Regex(@"^\s*Now listening on: (?<url>.*)$", RegexOptions.None | RegexOptions.Compiled, TimeSpan.FromSeconds(10));
+        private static readonly Regex NowListeningRegex = new Regex(@"\s*Now listening on: (?<url>.*)$", RegexOptions.None | RegexOptions.Compiled, TimeSpan.FromSeconds(10));
```

Comment:
To be honest I think we could just drop the `\s` here as well so it doesn't match the whitespace.

Reply:
Seems like it. Should it end in '\s*$' maybe?

Reply:
Makes sense to me.

Reply:
Updated

Reply:
This change looks good. Looking back at the implementation, I think we might benefit from having some sort of limit on the number of lines that this regex is evaluated on before watch bails. This change makes it slightly worse in terms of perf, but I think we should be good to get this in. 

Note we have some tests for this here: https://github.com/dotnet/sdk/blob/master/src/Tests/dotnet-watch.Tests/BrowserLaunchTests.cs

Reply:
Thanks @pranavkm and @danmoseley!